connection_test.go

```diff
@@ -970,3 +970,49 @@ func TestPeerStatusChangeServer(t *testing.T) {
 	})
 	assert.Len(t, changes, 0, "unexpected peer status changes")
 }
+
+func TestContextCanceledOnClientTCPClose(t *testing.T) {
+
+	opts := testutils.NewOpts().DisableLogVerification().NoRelay()
+
+	testutils.WithTestServer(t, opts, func(server *testutils.TestServer) {
+
```

Comment:
nit: delete blank. In general, we don't do blank lines after a `{` or before a `}`